Cartfile

```diff
@@ -1,2 +1,2 @@
 github "raphaelmor/Polyline" ~> 4.2
-github "mapbox/turf-swift" ~> 0.5
+github "mapbox/turf-swift" ~> 0.5
```

Comment:
Could you please add newline characters at the end of touched files?
<img width="358" alt="Screen Shot 2020-07-09 at 16 54 04" src="https://user-images.githubusercontent.com/6073510/87048764-d0629200-c204-11ea-872a-7124674e4f56.png">

Reply:
Fixed.